server/index.js

```diff
@@ -64,17 +69,17 @@ router.get('/gamereviews/:gameid', async (req, res) => {
   }
 });
 
-// router.get('/reviewcount/:gameid', (req, res) => {
-//   fetch(`http://ec2-54-185-79-51.us-west-2.compute.amazonaws.com:3002/api/reviewcount/${req.params.gameid}`)
-//     .then(response => response.json())
-//     .then(results => {
-//       res.status(200).json(results);
-//     })
-//     .catch(e => {
-//       console.error(e);
-//       res.status(500).json({ error: 'Error fetching review counts' });
-//     });
-// });
+router.get('/reviewcount/:gameid', (req, res) => {
```

Comment:
I'm guessing this is in anticipation of server migration? That server is unlikely to be actually running right now, right?

Reply:
yeah that server is down, just wanted to see if it was still responding